src/structure/spatial.jl

```diff
@@ -167,11 +170,11 @@ function dihedralangle(vec_a::Vector{Float64},
         dot(cross(vec_a, vec_b), cross(vec_b, vec_c)))
 end
 
-
 """
-Calculate the omega angle in radians for an `AbstractResidue`. Arguments are the
-residue (atoms "N" and "CA" required) and the previous residue (atoms "CA" and
-"C" required).
+Calculate the omega angle in radians for an `AbstractResidue`. Arguments can
+either be a residue and the previous residue or a chain and the position as
+an integer" The first residue (or one at the given index) requires the
```

Comment:
Should be "Arguments can either be a residue and the previous residue or a chain and the position as a residue ID. The first..."